lighthouse-core/audits/accessibility/meta-refresh.js

```diff
@@ -11,6 +11,21 @@
  */
 
 const AxeAudit = require('./axe-audit');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of an accesibility audit that evaluates if the page uses a meta tag that refreshes the page automatically as use of this feature can frustrate users. This descriptive title is shown to users when no user action is required. */
```

Comment:
Title of an accesibility audit that evaluates if the page uses a meta tag that refreshes the page automatically. ~~as use of this feature can frustrate users.~~

Reply:
done